From 6310d10305121769cc5c9ca6e4fa18939ed98017 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Tue, 24 Feb 2026 15:15:54 +0100
Subject: [PATCH] base-files: ssh-copy-id compatibility symlink

Program ssh-copy-id is broken and whenever it finds openwrt_release file
it expects that the distribution is using dropbear and weird setup of
ssh keys. This should make it more compatible.
---
 package/base-files/files/etc/dropbear/.authorized_keys | 1 +
 1 file changed, 1 insertion(+)
 create mode 120000 package/base-files/files/etc/dropbear/.authorized_keys

diff --git a/package/base-files/files/etc/dropbear/.authorized_keys b/package/base-files/files/etc/dropbear/.authorized_keys
new file mode 120000
index 0000000000..6e3825256b
--- /dev/null
+++ b/package/base-files/files/etc/dropbear/.authorized_keys
@@ -0,0 +1 @@
+../../root/.ssh/authorized_keys
\ No newline at end of file
-- 
2.53.0

